src/Benchmarks/Middleware/MultipleQueriesEfMiddleware.cs

```diff
@@ -21,20 +22,19 @@ public class MultipleQueriesEfMiddleware
         };
 
         private readonly RequestDelegate _next;
-        private readonly EfDb _db;
 
-        public MultipleQueriesEfMiddleware(RequestDelegate next, EfDb db)
+        public MultipleQueriesEfMiddleware(RequestDelegate next)
```

Comment:
It appears the changes in this file significantly increase the load on SQL, which causes SQL to be the bottleneck instead of the web server.